modules/router/manifests/nginx.pp

```diff
@@ -76,9 +76,9 @@
   $check_requests_warning = '@25',
   $check_requests_critical = '@10',
   $robotstxt = '',
-  $proxy_busy_buffers_size = '8k',
-  $proxy_buffers = '8 4k',
-  $proxy_buffer_size = '4k',
+  $proxy_busy_buffers_size = '16k',
```

Comment:
Hi Ian, really keen for this to get in but I think this will only work for Router machines. I'd like to see it done for all apps but I'm not sure where that would go but the problem you mentioned is also occurring in Signon and I don't think this would solve that issue.

Reply:
Completely agree we should put this on all machines. I'm very much fumbling in the dark with this change, and I'm unsure about were would this change be made to ensure that Signon etc wasn't left out?

Reply:
yeah me too, I'll ask around a bit :)